src/Agent.Sdk/Util/PlatformUtil.cs

```diff
@@ -171,18 +190,23 @@ private static string GetLinuxId()
                         }
                     }
                 }
+
+                throw new Exception($"Can't find ID in {filePath}");
```

Comment:
Will this exception be visible for the user? If so, we could use loc strings
+, consider making more concrete exception type ([Predefined](https://learn.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#use-the-predefined-net-exception-types) or [custom one](https://learn.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#end-exception-class-names-with-the-word-exception))